Makefile

```
@@ -65,6 +72,21 @@ compile-docker:	 	## Bundles Infection into a PHAR using docker
 compile-docker: $(DOCKER_FILE_IMAGE)
 	$(DOCKER_RUN_82) make compile
 
+.PHONY: sbx-image-build
+sbx-image-build:		## Builds the PHP sbx image
```

checkmake (0.3.2)

[warning] Target body for "sbx-image-build" exceeds allowed length of 5 lines (8).

(maxbodylength)